File: src/main/java/com/rs/java/game/area/banks/DuelArenaBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.Shape;
import com.rs.java.game.area.shapes.Polygon;

public final class DuelArenaBank extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Duel arena Bank";
	}

	@Override
	public Shape[] shapes() {
		return new Shape[] { new Polygon(new WorldTile[] {
				new WorldTile(3380, 3267, 0),
				new WorldTile(3380, 3273, 0),
				new WorldTile(3385, 3273, 0),
				new WorldTile(3385, 3267, 0) }) };
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/DuelArenaBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon

class DuelArenaBank : Area(
    Polygon(
        arrayOf(
            WorldTile(3380, 3267, 0),
            WorldTile(3380, 3273, 0),
            WorldTile(3385, 3273, 0),
            WorldTile(3385, 3267, 0)
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Duel arena Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
